src/index.js

```diff
@@ -13,7 +13,7 @@ const VLazyImageComponent = {
     },
     intersectionOptions: {
       type: Object,
-      default: {}
+      default: () => {}
```

Comment:
This doesn't actually returns an object, but `undefined` instead, since the curly braces on arrow functions indicate the opening/closing of a function